public/stylesheets/booking.css

```diff
@@ -0,0 +1,2 @@
+.summary{display:flex;justify-content:space-around;flex-grow:1}.summary>*{flex-grow:1;width:50%}.summary .dep-summary{background:#f1f1f1}.summary .arr-summary{background:#e2e2e2}.summary p{font-size:2em;text-align:center;font-style:italic;font-weight:900}
+/*# sourceMappingURL=booking.css.map */
```

Comment:
What is this?

Reply:
That's what I was struggling with yesterday. Shouldn't sass be compiled to minified css?